speedcrunch-git/.SRCINFO

```diff
@@ -0,0 +1,18 @@
+pkgbase = speedcrunch-git
```

Comment:
Surely one of the calculators that are available in the Arch repos will suffice, no?